src/Explorer/Panes/__snapshots__/PublishNotebookPaneComponent.test.tsx.snap

```diff
@@ -52,28 +52,29 @@ exports[`PublishNotebookPaneComponent renders 1`] = `
     <StackItem>
       <StyledWithResponsiveMode
         ariaLabel="Cover image"
-        defaultSelectedKey="URL"
+        defaultSelectedKey="Custom Image"
         label="Cover image"
         onChange={[Function]}
         options={
           Array [
-            Object {
-              "key": "URL",
-              "text": "URL",
-            },
             Object {
               "key": "Custom Image",
```

Comment:
>key [](start = 15, length = 3)

nt: why key should have space in its name?

Reply:
it can have a space, we are using the same enum for both key and display text.

Reply:
it can have a space, we are using the same enum for both key and display text.